openapi.yaml

```diff
@@ -280,14 +280,6 @@ paths:
           schema:
             type: string
             format: uri
-        - name: flow_type
-          in: query
-          description: (Optional) OAuth flow used for authorization. When not specified, `implicit` is used. Given that `pkce` is a more secure option, it is recommended that users of this API use `pkce`.
-          schema:
-            type: string
-            enum:
-              - pkce
-              - implicit
```

Comment:
Technically this is a breaking change, but since it's not fully released yet is OK. We need to prevent this from happening by focusing more on the RFC process.

Reply:
i think it might have been mentioned during the RFC but it wasn't obvious that it was unnecessary and it was only spotted during the implementation phase